### DISCUSSION TITLE
Media Source speed higher than 200% possible?

### DISCUSSION THREAD

**Discussion by fooryo** [obsproject/obs-studio#10611] [General]
As title: is it possible (maybe via websocket or any other API obs has) to set the Media source playback speed to values higher than 200% ?

OS: Archlinux
OBS-Studio 30.1.1-2

Thanks

**Comment by Fenrirthviti** [comment]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

As a note, the 200% limit was chosen because we had to have one with the way Qt operates. There's no specific reason otherwise than 200% would likely cover most use cases for non-1x playback.

Thank you!
